src/bindings/python/CMakeLists.txt

```diff
@@ -39,6 +46,10 @@ set(_packages_path "${_path}/Lib/site-packages")
 unset(_path)
 endif()
 message(STATUS "Installing python modules to ${CMAKE_INSTALL_PREFIX}/${_packages_path}")
-install(TARGETS python_valhalla
+install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/valhalla
```

Comment:
install the `python_valhalla.so` also in `<py_dir>/valhalla`. will be important later, when wheels will be built, where it'll need to pull all non-standard dynamic libs next to the valhalla.so, which then should happen in its own namespace..